src/tests/src/libs/antares/study/parts/hydro/test-hydroreader-class.cpp

```diff
@@ -166,14 +164,14 @@ BOOST_FIXTURE_TEST_CASE(Testing_support_for_old_studies, Fixture)
     auto& genE = area_1->hydro.dailyNbHoursAtGenPmax[0];
     auto& pumpE = area_1->hydro.dailyNbHoursAtPumpPmax[0];
 
-    auto& genPReader = reader->dailyMaxPumpAndGen[HydroMaxTimeSeriesReader::genMaxP];
-    auto& pumpPReader = reader->dailyMaxPumpAndGen[HydroMaxTimeSeriesReader::pumpMaxP];
-    auto& genEReader = reader->dailyMaxPumpAndGen[HydroMaxTimeSeriesReader::genMaxE];
-    auto& pumpEReader = reader->dailyMaxPumpAndGen[HydroMaxTimeSeriesReader::pumpMaxE];
+    auto& genPReader = dailyMaxPumpAndGen[0U];
+    auto& pumpPReader = dailyMaxPumpAndGen[2U];
+    auto& genEReader = dailyMaxPumpAndGen[1U];
+    auto& pumpEReader = dailyMaxPumpAndGen[3U];
```

Comment:
Use enum ?

Reply:
Enum is now private, I can't use it outside of the class.

Reply:
I think it would be best to keep it public

Reply:
This is @guilpier-code comment about this. https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1723#discussion_r1381500307

Reply:
Done!